### ISSUE TITLE
[GRPC] incorrect error message is returned while creating logical port with incorrect physical port number

### ISSUE THREAD

**Issue by andriidovhan** [telstra/open-kilda#3973]
**Step to reproduce:**
* try to create a logical port with an incorrect physical port
```
{
  "type" : "LAG",
  "port_numbers" : [ 44444 ],
  "logical_port_number" : 1111
}
```

**Expected result:**
The `"Invalid portno value."` error message should be returned.

**Actual result:**
The `"Provided logical port does not exist."` error message is returned.

**Comment by vkhalin** [comment]
fixed in #4725
uncommented in #4780

**Comment by vkhalin** [comment]
waiting for test code merge to close this ticket
